### DISCUSSION TITLE
Consentement ?

### DISCUSSION THREAD

**Discussion by olivvv59** [MyElectricalData/myelectricaldata_import#421] [Q&A]
Bonjour,

Le consentement est il à nouveau opérationnel ? 
ou c'est moi qui me débrouille pas bien ? 
je tombe sur une page 
```
{
  "reasons" : [ ],
  "details" : {
    "msgId" : "Id-XXXXXXXXXXXXXXXXXXX"
  }
}
```

j'ai essayé de changer dans l'url l'adresse de redirection mais sans succès. 

Merci pour votre réponse :-)

**Comment by m4dm4rtig4n** [comment]
Service tjr HS chez Enedis.

**Reply by grrosminet** [reply]
Peut-être faudrait-il modifier le message de la page d'accueil qui faisait rêver à un retour le 25 sept. et maintenant fait paniquer ? ;-)

**Comment by bolderiz** [comment]
A ce jour j'ai un autre résultat. Est ce que c'est normal ? 
![Capture](https://github.com/MyElectricalData/myelectricaldata_import/assets/25219711/a05f05ea-6fee-4f66-bb22-710a4678af1d)

**Reply by m4dm4rtig4n** [reply]
Soucis chez Enedis, ta une procédure temporaire sur la homepage pour fix le soucis.

**Reply by bolderiz** [reply]
Effectivement ca a beau être rouge. Il y a trop de texte je suis passé à coté
